tests/testthat/test-survival-tune-race.R

```diff
@@ -22,19 +25,24 @@ test_that("race tuning survival models with static metric", {
   split <- initial_split(sim_dat)
   sim_tr <- training(split)
   sim_te <- testing(split)
-  sim_rs <- vfold_cv(sim_tr)
+  sim_rs <- bootstraps(sim_tr, times = 20)
```

Comment:
do we need 20 bootstraps?

Reply:
Sure. I wanted to simulate a case where it got down to a single configuration and more bootstraps helps make that happen.